plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/AccountSelector.java

```diff
@@ -107,30 +95,35 @@ public String getSelectedEmail() {
     return combo.getText();
   }
 
+  /** @exception IllegalArgumentException if there is no account logged in */
+  public String getFirstEmail() {
+    Preconditions.checkArgument(getAccountCount() > 0);
```

Comment:
Gosh, I keep slipping this. I prefer the precondition, because no one in our code calls (and should not call) this method while in a broken state.